src/main/views/subscription-search-case-results.njk

```diff
@@ -0,0 +1,34 @@
+{% extends "template.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "./macros/common-components.njk" import goBack, submitButton %}
+
+{% block pageTitle %}
+  {{ "Subscription Case Search Results" | safe }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ goBack('/subscription-case-search') }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">Search result</h1>
+  <h3 class="govuk-heading-m">{{ searchResults.length }} result successfully found</h3>
+
+  <table class="govuk-table govuk-!-width-two-thirds">
+    <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header">Case reference number</th>
+      <th scope="col" class="govuk-table__header">Case name</th>
```

Comment:
The second column I think should be right aligned (alongside the text within it)